test_conformance/c11_atomics/test_atomics.cpp

```diff
@@ -109,6 +131,15 @@ int test_atomic_store_generic(cl_device_id deviceID, cl_context context,
         TYPE_ATOMIC_DOUBLE, useSVM);
     EXECUTE_TEST(error,
                  test_double.Execute(deviceID, context, queue, num_elements));
+
+    if (gFloatAtomicsSupported)
+    {
+        CBasicTestStore<HOST_ATOMIC_HALF, HOST_HALF> test_half(TYPE_ATOMIC_HALF,
+                                                               useSVM);
+        EXECUTE_TEST(error,
+                     test_half.Execute(deviceID, context, queue, num_elements));
+    }
```

Comment:
This is mostly cosmetic, but in the `CBasicTest` destructor, we're currently printing a message similar to `138 tests executed successfully for atomic_half`.  For most of the atomic types, this gets printed at the end of this function, when the destructors execute for the other tests, but for `atomic_half` it gets printed midway through execution.

Actually, the prints for e.g. `atomic_ptrdiff_t` is a little weird, too, since it gets printed when it goes out of scope, before e.g. `atomic_double`.

Given the weirdness for the other types, I'm inclined not to worry about this, but mentioning it for completeness, in case anyone else wonders where the print is for `atomic_half`.